jep/211/README.adoc

```diff
@@ -405,10 +405,31 @@ The following backward compatibility requirements are defined:
 
 == Security
 
-* Only Java 11 with the latest security fixes will be supported at the moment of the public release
-* Starting from the Java 11 preview availability,
-  Jenkins security issues on Java 11 will be processed according to the
-  standard link:https://jenkins.io/security/[Jenkins Security Process]
+=== Process
+
+Only Java 11 with the latest security fixes will be supported at the moment of the public release
+
+Starting from the Java 11 preview availability,
+we adopt a partial security process for Java 11 issues.
+Approach:
+
+* Security issues for Java 11 should be always reported according
+  to the link:https://jenkins.io/security/#reporting-vulnerabilities[standard vulnerability reporting process].
+  It is mandatory, because it is required potential impact on Java 8 users
+* Security issues for Java 11 will be triaged by the _Java 11 Support Team_,
+  the team should include Security team members
+* Security fixes for Java 11 Preview may be delivered directly to the Core and Plugins in public,
```

Comment:
i.e. "fixes for vulnerabilities on Java 11, but that are *not* vulnerabilities when using a Java 8 runtime"

Reply:
Could you please propose a change? On the phone